tests/test_search.py

```diff
@@ -0,0 +1,37 @@
+from jupyddl.heuristics import goal_count_heuristic, zero_heuristic
+from jupyddl.automated_planner import AutomatedPlanner
+from jupyddl.dijkstra import DijkstraBestFirstSearch
+from jupyddl.a_star import AStarBestFirstSearch
+from jupyddl.bfs import BreadthFirstSearch
+from jupyddl.dfs import DepthFirstSearch
+from os import path
+import coloredlogs
+import sys
+
+apla = AutomatedPlanner("data/domain.pddl", "data/problem.pddl")
+
+
+def test_searchDFS():
+    dfs = DepthFirstSearch(apla)
+    _, ct = dfs.search()
+    assert ct != 0
+
+
+def test_searchBFS():
+    bfs = BreadthFirstSearch(apla)
+    _, ct = bfs.search()
+    assert ct != 0
+
+
+def test_searchDijkstra():
+    dijk = DijkstraBestFirstSearch(apla)
+    res = dijk.search()  # None, computation_time, opened_nodes(in this order)
```

Comment:
Careful here, None is returned in the first element only if the path wasn't found, otherwise it returns the last node object (which should be the goal)